Makefile

```diff
@@ -24,7 +24,7 @@ TOOLS = \
 VERSION = $(shell git describe --tags --dirty --exact-match 2>/dev/null || git rev-parse --short HEAD)
 
 GO_LDFLAGS = \
-	-ldflags "-X main.Version=$(VERSION)"
+	-ldflags "-X main.Version=$(VERSION) -linkmode=internal -s -w"
```

Comment:
Can you explain why this is needed? I think this won't work on Yocto, because Yocto needs a standard ELF binary with a symbol table. There are ways around it, but let's make sure it's needed first.

Reply:
Ah you're right, that seems to be left from before I moved these to the separate windows flags, thanks! I removed the corresponding part...